.config/.lintstagedrc.js

```diff
@@ -9,6 +9,8 @@ export default {
 	// And elsewhere we don't, compare to https://github.com/stylelint/stylelint/pull/8009
 	'*.{css,scss}': 'stylelint --fix --allow-empty-input --no-validate',
 	'*.{js,ts,tsx,jsx,mjs,cjs}': 'xo --fix',
-	// ensure that security vulnerabilities are fixed before committing
-	'package-lock.json': 'npm audit fix --omit=dev'
+	// ensure that security vulnerabilities are fixed before committing - we need to skip `dev` for the moment as there are some unsolveable conflicts
+	'package-lock.json': 'npm audit fix --omit=dev',
```

Comment:
```suggestion
    '**/package-lock.json': 'npm audit fix --omit=dev',
```
Might be necessary to also reach lockfiles in subfolders (workspaces).

Reply:
we only do have a `package-lock.json` file in the root folder. And defining a file like `'package-lock.json'` would still include files in subfolders with the same name.

> If the glob pattern contains no slashes (`/`), `micromatch`'s matchBase option will be enabled, so globs match a file's basename regardless of directory: https://www.npmjs.com/package/lint-staged#user-content-filtering-files